docs/content/components/breadcrumb.mdx

```diff
@@ -155,32 +155,6 @@ https://designsystem.digital.gov/components/breadcrumb/
 
 <TabItem displayText="HTML" id="html">
 
-<h3>HTML Example Component</h3>
-
-<h3 className="site-preview-heading">Default breadcrumb</h3>
-<nav className="usa-breadcrumb" aria-label="Breadcrumbs,,">
-	<ol className="usa-breadcrumb__list">
-		<li className="usa-breadcrumb__list-item">
-			<a href="example/path" className="usa-breadcrumb__link">
-				<span>Home</span>
-			</a>
-		</li>
-		<li className="usa-breadcrumb__list-item">
-			<a href="example/path" className="usa-breadcrumb__link">
-				<span>Federal Contracting</span>
-			</a>
-		</li>
-		<li className="usa-breadcrumb__list-item">
-			<a href="example/path" className="usa-breadcrumb__link">
-				<span>Contracting assistance programs</span>
-			</a>
-		</li>
-		<li className="usa-breadcrumb__list-item usa-current" aria-current="page">
-			<span>Women-owned small business federal contracting program</span>
-		</li>
-	</ol>
-</nav>
-
 <Code className="html">{`
 <h3 class="site-preview-heading">Default breadcrumb</h3>
 <nav class="usa-breadcrumb" aria-label="Breadcrumbs,,">
```

Comment:
I know it's not you but just noticed that  aria-label

Reply:
This appears to have been done by USWDS as an accessibility fix for the pages with breadcrumb examples. I think the best approach may be to make them better than just `breadcrumb,,,,,,,,`.

Reply:
@olitharp-nci and @karlikpj -- this is actually a byproduct of accessibility scanning. I don't like the commas, we should give nice neat labels so people know what they are looking at and provide guidance on what it should actually be on a page. (Breadcrumbs has the guidance)

Reply:
Oh, @arcepaul TIL'd that labels need to be unique